### PR TITLE
Implemented graph using ArrayList in java

### PR DESCRIPTION
**Pull request by PriyankaPatel1612**
### Description
Implemented graph using ArrayList

### Related/Fixes Issue
Issue #67

### Screenshots (if applicable)

- [x ] Fixed Bug/issues
- [x] Added algo/questions_answers/docs
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Comment by rohan472000** [comment]
image
fix the first checkbox.

File: Graphs/GraphUsingArrayList.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+
+
+//Implementing graph using java
+//Implemented adjacency list using arraylist
+//for undirected unweighted graph
+
```
Comment (rohan472000) [inline Graphs/GraphUsingArrayList.java:6]:
```suggestion
//Implementing graph using java
//Implemented adjacency list using arraylist
//for undirected unweighted graph

```

Comment (rohan472000) [inline Graphs/GraphUsingArrayList.java:6]: Add wikipedia link here.

File: Graphs/GraphUsingArrayList.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        graph.get(d).add(s); 
+    } 
+
+    static void printAdjacencyList(ArrayList<ArrayList<Integer> > graph) 
+    { 
+        for (int i = 0; i < graph.size(); i++) { 
+            //Printing the vertex
+            System.out.println("Adjacency list of " + i); 
+            for (int j = 0; j < graph.get(i).size(); j++) { 
+                //printing neighbours of vertices
```
Comment (rohan472000) [inline Graphs/GraphUsingArrayList.java:27]:
```suggestion
                //printing neighbours vertices
```

**Review by rohan472000 — CHANGES_REQUESTED** [review]

File: Graphs/GraphUsingArrayList.java (hunk 1 above)
Comment (PriyankaPatel1612) [reply Graphs/GraphUsingArrayList.java]: Okay.

File: Graphs/GraphUsingArrayList.java [hunk 3]
```diff
@@ -1,3 +1,6 @@
+
```
Comment (rohan472000) [inline Graphs/GraphUsingArrayList.java:1]:
```suggestion
```

**Review by rohan472000 — CHANGES_REQUESTED** [review]

**Comment by rohan472000** [comment]
Thanks for your contribution.

**Comment by PriyankaPatel1612** [comment]
Thank you for merging my pull request.

**Pull request merged**